### PR TITLE
[19-10-2022] Event English and Arabic subpages

### PR DESCRIPTION
**Pull request by adetyaz**
Summary: https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/186
Tasks 3
We need to add an events subpage where we can link our events in [Zenodo](https://zenodo.org/communities/1231231664/?page=1&size=20).
3.1 Add this to the nav bar
3.2 Add three cards with each event, title, name of the speaker, and language.
3.3 Each card should direct you to a subpage with a description of the event, speaker, language and a link to the slides from Zenodo.

Fixes
Event subpages open when event card is clicked 

Screenshot: 
event details

### CODE REVIEW

**Comment by sandygudie** [comment]
Welldone @adetyaz this is good,👏 and the arrangement is great, a few suggestions
- Add a description Heading title
- Include a link to view its slides
- Use the sub-page format, the sample file is in the `eng` folder
- increase the image size a bit
others
- the title of your PR should be like this [Date] Task3.2....

**Review by sandygudie — COMMENTED** [review]
Welldone @adetyaz this is good,👏 and the arrangement is great, a few suggestions
- Add a description Heading title
- Include a link to view its slides
- Use the sub-page format, the sample file is in the `eng` folder
- increase the image size a bit
others
- the title of your PR should be like this [Date] Task3.2....

**Comment by adetyaz** [comment]
@sandygudie thanks, I'll get on it.

I added a link to the slides page with continue reading, do you want that to be modified in anyway?

I also don't understand what you mean by a description heading title, please expantiate. thanks

**Comment by adetyaz** [comment]
@sandygudie I also mentioned I need the arabic translation for the arabic language event subpages, please how do I go about it.

File: event-pages/event-1.html [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+						<div class="section-title-divider"></div>
+					</div>
+					<div class="event-body">
+						<img
+							src="../assets/img/event/event1.png"
+							alt="event-image"
+							class="event-image"
+						/>
+						<div class="mt-5">
+							<p class="event-description">
```
Comment (sandygudie) [inline event-pages/event-1.html:149]: Add a "Description"  text before the describtion content`<h4> Description <h4/>`

Comment (adetyaz) [reply event-pages/event-1.html:149]: understood

**Comment by sandygudie** [comment]
> @sandygudie I also mentioned I need the arabic translation for the arabic language event subpages, please how do I go about it.

The Arabic translation is the equivalent page but written in Arabic, (you can use google ) 
Example: if you toggle this button on the home page you will see the Arabic version of the homepage 
image
The Arabic page of the home page is the` ar-index.html` file in the project folder.

 So for the event sub-page, every event page will have its `ar-event1.html ` etc
you will have a toggle button on the navbar that will toggle to show its` ar-event1.html` page

**Comment by sandygudie** [comment]
> @sandygudie thanks, I'll get on it.
> 
> I added a link to the slides page with continue reading, do you want that to be modified in anyway?
> 
> I also don't understand what you mean by a description heading title, please expantiate. thanks

The **continue reading** is a link to the Zenode page, the link to the slide pages is something like this :https://docs.google.com/presentation/d/e/2PACX-1vRgYS2oCeD9PV3rPTMu9ElipG8K1R02KPXpcr01NatLaMC1VI8Jc_-5u72PB05HiDNUm0YC5ZjVCk8_/pub?start=false&loop=false&delayms=3000&slide=id.p
Hence user can view the slides pages directly without going to Zenedo

**Comment by adetyaz** [comment]
@sandygudie hello , I have made the changes asked of me, the second event doesn't have a direct link to the slide so it still linksto the zenodo page

**Comment by sandygudie** [comment]
> @sandygudie hello , I have made the changes asked of me, the second event doesn't have a direct link to the slide so it still linksto the zenodo page

alright @adetyaz I will review

**Comment by sandygudie** [comment]
@adetyaz, Welldone on the Arabic page. I made a sample page of what I have in mind;
image
image
-And feel free to add more content to the page

**Comment by adetyaz** [comment]
@sandygudie Thanks
Alright,will make the changes

**Comment by adetyaz** [comment]
@sandygudie I have made the changes

**Pull request closed without merging**